src/components/Button.js

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 const Button = props => {
   const { name } = props;
   return (
-    <button type="button">
+    <button type="button" className="btn">
       {name}
```

Comment:
- [ ] Button component should accept `color` and `wide` as incoming props.
![Screenshot from 2021-01-13 12-48-25](https://user-images.githubusercontent.com/30977845/104475256-a5833d00-559d-11eb-9f2f-dc58f58beb7e.png)

- [ ] Default `color` for the button should be orange.
![Screenshot from 2021-01-13 12-49-46](https://user-images.githubusercontent.com/30977845/104475413-d5cadb80-559d-11eb-8a3e-f97dfb659e27.png)

- [ ] `wide` is boolean, true means that button take 50% of space (used for `0` button)
![Screenshot from 2021-01-13 12-50-39](https://user-images.githubusercontent.com/30977845/104475507-f430d700-559d-11eb-9825-d07f8b16a1a5.png)